spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ExampleRequest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.spring;
+
+import javax.validation.constraints.NotNull;
+
+public final class ExampleRequest {
+    private final String serviceType;
+    private final String methodName;
+    private final Object exampleRequest;
+
+    private ExampleRequest(String serviceType, String methodName, Object exampleRequest) {
+        this.serviceType = serviceType;
+        this.methodName = methodName;
+        this.exampleRequest = exampleRequest;
+    }
+
+    public String getServiceType() {
+        return serviceType;
+    }
+
+    public String getMethodName() {
+        return methodName;
+    }
+
+    public Object getExampleRequest() {
+        return exampleRequest;
+    }
+
+    public static ExampleRequest of(@NotNull String serviceType,
```

Comment:
- Could you move this method to the top of the class so that public static methods come before member fields/methods?
- Could you add `toString()`?

Reply:
Thank you for review. I fixed it in 0f88123.